geoPipe.py

```diff
@@ -436,21 +535,110 @@ def main():
 	print("\ndetermining trail shape")
 	trail_df = trail_df.apply(util.is_loop, args=(LOOP_COMPLETION_THRESHOLD_METERS, ), axis=1)
 
-
 	# 14. Find bus stops
-	print("\nfinding bus stops")
-	trail_df = trail_df.progress_apply(util.get_bus, args=(BUS_RADIUS_METERS, ), axis=1)
+	#print("\nfinding bus stops")
+	#trail_df = trail_df.progress_apply(util.get_bus, args=(BUS_RADIUS_METERS, ), axis=1)
 
 	# print(trail_df.columns)
 	# print(trail_df)
 
 	# 15. Insert trails into database
+	print("\nUploading to DB")
 	tablename = 'destinations_osm'
 	schema = 'public'
-
 	to_db(trail_df, region_code, tablename, schema)
 
-	print("found " + str(len(trail_df)) + " trails in " + REGION)
+	return len(trail_df)
+	
+def get_region_code(park_id, park_index):
+	return TYPE + "_" + park_id + "_" + str(park_index)
+
+def main():
+
+	sys.setrecursionlimit(10000)
```

Comment:
Wait where are you using recursion in this code?

Reply:
ways_to_trails method is recursive, but this was code from before, I did not add any new recursion in this PR, just moved methods around.